core/templates/components/graph-services/graph-layout.service.ts

```diff
@@ -485,7 +485,7 @@ export class StateGraphLayoutService {
     return nodeData;
   }
 
-  getLastComputedArrangement(): NodeDataDict {
+  getLastComputedArrangement(): NodeDataDict | null {
     return cloneDeep(this.lastComputedArrangement);
```

Comment:
This is `null` because we are initializing `lastComputedArrangement` also as `null`. 
**Refer Line: 76**

Reply:
Also, inside **.spec.ts** file we are expecting it to be `null` before running `computeLayout()` function.
**Refer Line: 764** 

![image](https://user-images.githubusercontent.com/75239620/144308348-d9677e8c-7e8b-4422-ad27-ff362180594b.png)

Reply:
Looks good. Please add an explanatory comment.

Reply:
Added one. Kindly review it